src/ontology/config/properties.txt

```diff
@@ -14,6 +14,7 @@ http://www.geneontology.org/formats/oboInOwl#inSubset
 http://www.geneontology.org/formats/oboInOwl#source
 http://www.geneontology.org/formats/oboInOwl#hasSynonymType
 http://www.geneontology.org/formats/oboInOwl#SynonymTypeProperty
+http://purl.obolibrary.org/obo/mondo#ABBREVIATION
```

Comment:
This needs to be `http://purl.obolibrary.org/obo/OMO_0003012`

Reply:
Yep! I'm on that; that is part of [abandoning](https://github.com/monarch-initiative/mondo-ingest/pull/93#discussion_r1716076923) the mondo:abbreviation at the components level.